doc/release-notes/0.14/0.14.md

```diff
@@ -112,6 +112,12 @@ information, see <a href="https://www.eclipse.org/openj9/docs/xshareclasses/">-X
 information, see <a href="https://www.eclipse.org/openj9/docs/xxjitinlinewatches/">-XX:[+|-]JITInlineWatches</a>. </td>
 </tr>
 
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/5156">#5156</a></td>
+<td valign="top">Change to the default native stack size</td>
```

Comment:
The option affected `-Xmso` isn't mentioned in the release note.

Reply:
Good point! I added a mention